src/CMakeLists.txt

```diff
@@ -206,7 +206,7 @@ target_link_libraries(slade
 	${LUA_LIBRARIES}
 	${MPG123_LIBRARIES}
 	${fmt_LIBRARIES}
-	-lstdc++fs
+
```

Comment:
Doesn’t it break linking on Linux?

Reply:
I have not tested on Linux, I was hopping someone would give me feedback about that.
Deleting it is necessary for it to work on MacOS.

Reply:
Ok, now it check if its macOS or not

Reply:
I cannot check right now, but is it a correct CMake syntax?